apps/site/src/content/blog/state-of-the-budgie-2023.mdx

```diff
@@ -0,0 +1,203 @@
+
+Budgie 10.8 was the first release to start requiring Magpie v0.x and the result has been a more consistent and reliable experience for our users.
+
+## Wayland efforts
+
+While our goal in 2023 was only to start experimenting with Wayland compositors and to take the learnings from those experiments to apply to a "future Budgie WM compositor" built for Budgie 11, we made a significant shift in our development focus starting middle of last year to accelerate our Wayland plans and work towards Budgie 10 being Wayland-only (and continuing to be Wayland-only going into Budgie 11). This was in recognition of several major developments in the broader ecosystem that we covered in our [Wayland](/wayland) post.
+
+In that lengthy post, we covered our experiences at that point with wlroots as well as XFCE's libxfce4windowing library, which provides us a library which supports both X11 (via libwnck) and Wayland (various Wayland protocols) at the same time. This is especially useful as we iterate on making Budgie Wayland-compatible while using it at the same time.
+
+I am happy to report that progress on Wayland support has been steady on all fronts since our shift in development focus to it.
+
+In Budgie Desktop, several applets and UX components of Budgie Desktop have been ported over or are in the process of being ported:
+
+- Show Desktop Applet has been ported
+- Tab Switcher has been ported
+- Workspaces Applet has been ported
+- IconTasklist Applet is currently being ported.
+
+Keep in mind that not every part of the codebase needs to be touched, but some key areas in the codebase will need to be updated once some newer window tracking logic has been merged in from the IconTasklist conversion: MenuManager (right-click menus), Panel Manager (manages panels), the legacy Tasklist needs to be rewritten, and various uses of Gdk.X11 APIs.
+
+On the Magpie v1.0 front, the progress by Campbell Jones (aka serebit) has been fantastic as well:
+
+- @serebit write things here please :)
+
+## 2024
+
+For the new year, our focus is on two "epics".
+
+### Budgie 10 Epic
+
+Our main epic is delivering Budgie 10 on Wayland. This does not necessarily mean everything will be feature complete, as we still need to more concretely and granularly define remaining work and research around that, but it will mean that it can be daily driven and consumable in some form.
+
+For Q1, our focuses are on:
+
+1. Delivering Budgie 10.9 (as early this month) featuring our initial work with libxfce4windowing, as well as our new requirement for using budgie-session for launching Budgie Desktop. This will include the existing applets and Tab Switcher that have been ported and should provide plenty of time for distribution packagers to get Budgie Desktop into their respective releases without running up against any freeze deadlines.
```

Comment:
```suggestion
1. Delivering Budgie 10.9 (as early as this month) featuring our initial work with libxfce4windowing, as well as our new requirement for using budgie-session for launching Budgie Desktop. This will include the existing applets and Tab Switcher that have been ported and should provide plenty of time for distribution packagers to get Budgie Desktop into their respective releases without running up against any freeze deadlines.
```